Cake.Recipe/Content/addins.cake

```diff
@@ -15,7 +15,7 @@
 #addin nuget:?package=Cake.Slack&version=0.12.0
 #addin nuget:?package=Cake.Transifex&version=0.7.0
 #addin nuget:?package=Cake.Twitter&version=0.9.0
-#addin nuget:?package=Cake.Wyam&version=1.7.1
+#addin nuget:?package=Cake.Wyam&version=1.7.2
```

Comment:
Should also update tool

Reply:
Are you saying it didn’t? If so I’ll need to review the improvement I published last week which was supposed to do exactly that.

Reply:
@Jericho Nope, was still only the addin updated

Reply:
You manually updated the tool before accepting the PR?

Reply:
Yes, I updated the tool and ammended the commit before merging.

Reply:
Ok got it. I was looking at history and only saw the one commit by the audit tool so I assumed it had correctly updated the tool as well.

Reply:
Yeah, unfortunately GitHub doesn't show history well on a PR in case of force pushes. I think only indication is that https://github.com/cake-contrib/Cake.Recipe/pull/273/commits/ad7f22d2468313b33daad07963d700f02caea83d has me and cake-contrib-bot as commiter